tools/tune/mfcc/setup_mfcc.m

```diff
@@ -105,7 +118,20 @@ function setup_mfcc(cfg)
 v = cfg.use_energy;                              [b8, j] = add_w8b(v, b8, j); % bool
 
 %% Export
-eq_tplg_write(fn, b8, 'DEF_MFCC_PRIV', 'Exported MFCC configuration');
+switch cfg.tplg_ver
+       case 1
+	       tplg_write(cfg.tplg_fn, b8, "DEF_MFCC_PRIV", ...
+			  "Exported with script setup_mfcc.m", ...
+			  "cd tools/tune/mfcc; octave setup_mfcc.m");
+       case 2
+	       tplg2_write(cfg.tplg_fn, b8, "mfcc_config", ...
+			   "Exported MFCC configuration", ...
+			   "cd tools/tune/mfcc; octave setup_mfcc.m");
+       otherwise
+	       error("Illegal cfg.tplg_ver, use 1 or 2.");
```

Comment:
use 1 for X or 2 for Y - more helpful :)

Reply:
Done!